Src/Plugins/Security/SEC101_043.NuGetCredentialsValidator.cs

```diff
@@ -73,19 +73,19 @@ internal static List<string> ExtractHosts(string hostXmlAsString)
         protected override IEnumerable<ValidationResult> IsValidStaticHelper(ref string matchedPattern,
                                                                              Dictionary<string, string> groups)
         {
-            if (!groups.TryGetNonEmptyValue("id", out string id) ||
-                !groups.TryGetNonEmptyValue("host", out string xmlHost) ||
-                !groups.TryGetNonEmptyValue("secret", out string secret))
+            if (!groups.TryGetNonEmptyValue("host", out string xmlHost) ||
+                !groups.TryGetNonEmptyValue("secret", out string xmlCredentials))
             {
                 return ValidationResult.CreateNoMatch();
             }
 
-            if (FilteringHelpers.LikelyPowershellVariable(secret))
+            if (FilteringHelpers.LikelyPowershellVariable(xmlCredentials))
```

Comment:
>LikelyPowershellVariable [](http://example.com/codeflow?start=33&length=24)

Shouldn't this check occur against the contents of each XML element rather than the bundle of XML itself? 

i.e., will we actually ever enter this no match code path?

Reply:
removed this powershell condition.